OpenSim/Simulation/Manager/Manager.h

```diff
@@ -189,12 +189,12 @@ class OSIMSIMULATION_API Manager
     DEPRECATED_14("Get the state's time using SimTK::State::getTime().")
     double getInitialTime() const;
     /** <b>(Deprecated)</b> Integrate to a specified finalTime using 
-        Manager::integrate(SimTK::State&, double). */
-    DEPRECATED_14("Integrate to a specified finalTime using Manager::integrate(SimTK::State&, double).")
+        Manager::integrate(double). */
+    DEPRECATED_14("Integrate to a specified finalTime using Manager::integrate(double).")
     void setFinalTime(double aTF);
     /** <b>(Deprecated)</b> Integrate to a specified finalTime using
-        Manager::integrate(SimTK::State&, double). */
-    DEPRECATED_14("Integrate to a specified finalTime using Manager::integrate(SimTK::State&, double).")
+        Manager::integrate(double). */
+    DEPRECATED_14("Integrate to a specified finalTime using Manager::integrate(double).")
     double getFinalTime() const;
```

Comment:
Does this have any affect anymore?

Reply:
I'm not sure the "deprecated" way of using the manager actually works anymore.

Reply:
You're right, these functions shouldn't really do what you expect since they just update internal variables `_ti` and `_tf`. Also, I'm pretty sure you're right that these deprecated functions and the "deprecated" way of using the manager should just be removed.